tests/Unit/Evolution/Systems/ScalarAdvection/BoundaryCorrections/Test_Rusanov.cpp

```diff
@@ -0,0 +1,70 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Framework/TestingFramework.hpp"
+
+#include <cstddef>
+
+#include "Evolution/Systems/ScalarAdvection/BoundaryCorrections/Factory.hpp"
+#include "Evolution/Systems/ScalarAdvection/BoundaryCorrections/Rusanov.hpp"
+#include "Evolution/Systems/ScalarAdvection/System.hpp"
+#include "Framework/SetupLocalPythonEnvironment.hpp"
+#include "Framework/TestCreation.hpp"
+#include "Helpers/Evolution/DiscontinuousGalerkin/BoundaryCorrections.hpp"
+#include "Helpers/Evolution/DiscontinuousGalerkin/Range.hpp"
+#include "NumericalAlgorithms/Spectral/Mesh.hpp"
+#include "NumericalAlgorithms/Spectral/Spectral.hpp"
+#include "Utilities/TMPL.hpp"
+#include "Utilities/TaggedTuple.hpp"
+
+namespace helpers = TestHelpers::evolution::dg;
+
+namespace {
+
+template <size_t Dim>
+void test_rusanov(const gsl::not_null<std::mt19937*> gen,
+                  const size_t num_pts) {
+  helpers::test_boundary_correction_conservation<ScalarAdvection::System<Dim>>(
+      gen, ScalarAdvection::BoundaryCorrections::Rusanov<Dim>{},
+      Mesh<Dim - 1>{num_pts, Spectral::Basis::Legendre,
+                    Spectral::Quadrature::Gauss},
+      {}, {});
+
+  helpers::test_boundary_correction_with_python<ScalarAdvection::System<Dim>>(
+      gen, "Rusanov",
+      {{"dg_package_data_u", "dg_package_data_normal_dot_flux_u",
+        "dg_package_data_abs_char_speed"}},
+      {{"dg_boundary_terms_u"}},
+      ScalarAdvection::BoundaryCorrections::Rusanov<Dim>{},
+      Mesh<Dim - 1>{num_pts, Spectral::Basis::Legendre,
+                    Spectral::Quadrature::Gauss},
+      {}, {});
+
+  const auto rusanov = TestHelpers::test_creation<std::unique_ptr<
+      ScalarAdvection::BoundaryCorrections::BoundaryCorrection<Dim>>>(
+      "Rusanov:");
+
+  helpers::test_boundary_correction_with_python<ScalarAdvection::System<Dim>>(
+      gen, "Rusanov",
+      {{"dg_package_data_u", "dg_package_data_normal_dot_flux_u",
+        "dg_package_data_abs_char_speed"}},
+      {{"dg_boundary_terms_u"}},
+      dynamic_cast<const ScalarAdvection::BoundaryCorrections::Rusanov<Dim>&>(
+          *rusanov),
+      Mesh<Dim - 1>{num_pts, Spectral::Basis::Legendre,
+                    Spectral::Quadrature::Gauss},
+      {}, {});
+}
+}  // namespace
+
+SPECTRE_TEST_CASE("Unit.ScalarAdvection.Rusanov", "[Unit][Evolution]") {
```

Comment:
Here, "Unit.ScalarAdvection.BoundaryCorrections.Rusanov".

Okay, I'm done now, I promise 🙂 ... you can squash all these directly.

Reply:
I see the `#.BoundaryCorrections.#` keyword is absent for the test names in other systems (e.g. Unit.NewtonianEuler.Hll)

Do you think it would be good to place another PR fixing all of those?

Reply:
I do think so, and I have a commit that does that already prepared 🙂. If you make this change here we should have uniform names everywhere!

Reply:
Ah, I see! Great